drake/bindings/python/pydrake/rbtree.py

```diff
@@ -2,6 +2,10 @@
 
 from ._pydrake_rbtree import *
 
+# TODO: Remove this direct import and ask users to import
+# pydrake.parsers.PackageMap instead
+from .parsers import PackageMap
```

Comment:
thanks for making it backward compatible.  im happy to update director code if/when you want to remove this.